pallets/thea/src/lib.rs

```diff
@@ -538,8 +693,11 @@ pub mod pallet {
 			tx_hash: sp_core::H256,
 			bit_map: u128,
 			bls_signature: [u8; 96],
-		) -> DispatchResult {
-			let relayer = ensure_signed(origin)?;
+			relayer_index: u8,
+			_signature: T::Signature,
```

Comment:
Why we need this arg if we are not using it?

Reply:
Yes, we are using the argument to validate the tx

Reply:
Same as [here](https://github.com/Polkadex-Substrate/Polkadex/pull/669#discussion_r1143054287)